hubgrep/cli_blueprint/import_data.py

```diff
@@ -139,8 +150,14 @@ def import_repos(new_table_name):
         exports = hoster_dict.get("exports_unified")
         if exports:
             export_url = exports[0]["url"]
+            created_at = exports[0]["created_at"]
             before = time.time()
-            fetch_hoster_repos(export_url, hoster, new_table_name)
+            repo_count = fetch_hoster_repos(export_url, hoster, new_table_name)
+            with TableHelper._cursor() as cursor:
+                cursor.execute(
+                    f"update hosting_service set repo_count = %s, export_timestamp = %s where id = %s",
+                    (repo_count, created_at, hoster.id),
+                )
```

Comment:
after adding repos to the "new" table, we update the metadata in the hosting service

Reply:
todo: move counting and update to rotate function